### PR TITLE
feat(react-components): Move Persona to unstable

### LINKED ISSUE #25004: Persona: Move to unstable

### PR DESCRIPTION
**Pull request by sopranopillow**
This PR moves Persona to unstable and adds disclaimers of it being unstable.

## Related Issue(s)

Fixes #25004

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/20794d09e7cfa4b1a4ea42412bfb8378d1e0b9f3'>20794d09e7cfa4b1a4ea42412bfb8378d1e0b9f3</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/Avatar-mount_base.html">1660</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/Avatar-mount.html">1683</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/Button-mount_base.html">1166</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/Button-mount.html">1185</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProvider-mount_base.html">1982</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProvider-mount.html">1978</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProviderWithTheme-mount_base.html">707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProviderWithTheme-mount.html">731</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">658</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">681</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">708</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">707</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/MakeStyles-mount_base.html">2265</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/MakeStyles-mount.html">2263</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/SpinButton-mount_base.html">3288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25008/perf-test-react-components/SpinButton-mount.html">3267</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-persona/README.md [hunk 1]
```diff
@@ -3,3 +3,29 @@
 **React Persona components for [Fluent UI React](https://react.fluentui.dev/)**
 
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+## STATUS: WIP 🚧
+
+These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
```
Comment (khmakoto) [inline packages/react-components/react-persona/README.md:9]: This message is being repeated twice here, you should remove the one in line 5.

File: packages/react-components/react-persona/src/stories/Persona/PersonaDescription.md [hunk 2]
```diff
@@ -1 +1,14 @@
 A Persona is a visual representation of a person or status that showcases an Avatar, PresenceBadge, or an Avatar with a PresenceBadge.
+
+<!-- Don't allow prettier to collapse code block into single line -->
+<!-- prettier-ignore -->
+> **⚠️ Preview components are considered unstable:**
+>
+> ```jsx
+>
+> import { AvatarGroup, AvatarGroupItem, AvatarGroupPopover } from '@fluentui/react-components/unstable';
```
Comment (khmakoto) [inline packages/react-components/react-persona/src/stories/Persona/PersonaDescription.md:9]:
```suggestion
> import { Persona } from '@fluentui/react-components/unstable';
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**